app/src/main/java/helium314/keyboard/latin/suggestions/SuggestionStripView.java

```diff
@@ -129,7 +128,7 @@ public StripVisibilityGroup(final View suggestionStripView,
         public void setLayoutDirection(final boolean isRtlLanguage) {
             final int layoutDirection = isRtlLanguage ? ViewCompat.LAYOUT_DIRECTION_RTL
                     : ViewCompat.LAYOUT_DIRECTION_LTR;
-            ViewCompat.setLayoutDirection(mSuggestionStripView, layoutDirection);
+            ViewCompat.setLayoutDirection(mSuggestionStripView, ViewCompat.LAYOUT_DIRECTION_LOCALE);
             ViewCompat.setLayoutDirection(mSuggestionsStrip, layoutDirection);
```

Comment:
You set `mSuggestionStripView` to follow layout direction of the system locale, but `mSuggestionsStrip` to follow the keyboard locale.
Without testing myself, I'm not sure what this actually does,. Can you elaborate on why?

Reply:
From my testing, 'mSuggestionStripView' is responsible for the toolbar overall direction. Changing the layout direction of 'mSuggestionsStrip' seemed to only affect the direction of the vertical bars '|' that separate each suggested word.
Nevertheless I think it may be a good idea to make them both follow the same direction like they previously did, so I changed that in the new commit.